_test/shared/noisify_test_rand.m

```diff
@@ -0,0 +1,65 @@
+classdef noisify_test_rand < handle
```

Comment:
As this is a shared RNG class it is not inherently a "noisify" specific function so that shouldn't be part of the class name or be included in the function comment.

This is a pseudo random number generator which returns an ordered, linear  series, as described in the comment further down: 
```
%The values generated are an ascending sequence of increment one in 
% the range [0:999]*1e-3.
```

Reply:
Well, you were the one who asked for it to go in the generic utilities location. Maybe it should go back to the noisify location.

Reply:
I was suggesting that it should be here, but should be renamed without 'noisify' in the name.

Reply:
Suggest you check your original comment